app/models/study.rb

```diff
@@ -286,6 +295,10 @@ class Study < ApplicationRecord # rubocop:todo Metrics/ClassLength
                 allow_blank: true
               }
 
+    validates :ebi_library_strategy, inclusion: { in: EBI_LIBRARY_STRATEGY_OPTIONS }
```

Comment:
Can these be nil or are these required for all studies? Do we need to add a conditional to these so they are just checked for ENA studies?